snippets/head.md

```diff
@@ -8,9 +8,16 @@ Returns the head of a list.
 Use `arr[0]` to return the first element of the passed array.
 
 ```js
-const head = arr => arr[0];
+const head = arr => arr && arr.length ? arr[0] : undefined;
 ```
 
 ```js
 head([1, 2, 3]); // 1
-```
\ No newline at end of file
+head(); // undefined
+head([]); // undefined
+head(null); // undefined
+head(undefined); // undefined
```

Comment:
Most of the above tests should be added as test cases in the related file and we should keep some of them here (filled array, empty array and `null` should suffice).